docs/ispc.rst

```diff
@@ -632,6 +632,20 @@ Updating ISPC Programs For Changes In ISPC 1.25.0
 
 ``avx512knl-x16`` target is deprecated and will be removed in future releases.
 
+ISPC language was extended to support ``__attribute__(())`` syntax for variable
+and function declarations. Two attributes are supported now: ``noescape`` and
+``address_space(N)``. The macro ``ISPC_ATTRIBUTE_SUPPORTED`` is defined if the
+ISPC compiler supports these attributes.
+
+``__attribute__((noescape))`` can be placed on a function parameter of a
```

Comment:
Agree. I would also add a couple of code examples because they illustrate the feature better:
```
 void example1(__attribute__((noescape)) uniform int *ptr) {
     // ptr cannot escape the function
 }

 uniform int * example2(__attribute__((address_space(1))) uniform int *x) {
     // x is in address space 1
     return x;
 }
```

Reply:
I have introduced the separate `Attributes` section with description and examples.